### ISSUE TITLE
Build reports success but actually failed

### ISSUE THREAD

**Issue by emartynov** [openbakery/gradle-xcodePlugin#198]
I followed tutorial and I got output:
```
xcodebuild: error: The flag -scheme is required when specifying -derivedDataPath.

BUILD SUCCESSFUL
```

What should I do?

**Comment by emartynov** [comment]
Added schema and it fixed the build

**Comment by emartynov** [comment]
I created pull request to update README. But the proper fix should be also to fail build in this situation.

**Comment by renep** [comment]
fixed with 0.11.5
